### PR TITLE
Feat/send email register

### PR DESCRIPTION
**Pull request by fodekar**
## Checklist before merging
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update

### CODE REVIEW

File: www/Core/Mailer.class.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        $link = "<a href='localhost/verify-account?key=".$activationCode."'>Click and Verify Email</a>";
+        $this->phpmailer = new PHPMailer();
+        $this->phpmailer->isSMTP();
+        $this->phpmailer->Host = 'smtp-relay.sendinblue.com';
+        $this->phpmailer->SMTPAuth = true;
+        $this->phpmailer->Port = 587;
+        $this->phpmailer->Username = 'chancelvyfodekar@gmail.com';
+        $this->phpmailer->Password = 'YPM3jWUcsxK5trGf';
+        $this->phpmailer->addAddress($email);
+        $this->phpmailer->setFrom('cdeft.lalg@gmail.com', 'Fodekar');
```
Comment (danielamoudokpo) [inline www/Core/Mailer.class.php]: Bouger les identifiants dans conf.inc.php

File: www/Controller/User.class.php [hunk 2]
```diff
@@ -59,24 +60,44 @@ public function update()
...
             $user = $user->findOne(['id' => $userId]);
 
             if ($user === false) {
                 header("Location: /users");
             } else {
-                $user->setFirstname($_POST['firstname']);
-                $user->setLastname($_POST['lastname']);
-                $user->setEmail($_POST['email']);
+                $mailer = new Mailer();
+                $user = new UserModel();
```
Comment (danielamoudokpo) [inline www/Controller/User.class.php:74]: Why?

File: www/Controller/User.class.php [hunk 3]
```diff
@@ -59,24 +60,44 @@ public function update()
...
-                $user->save();
+                        $user->setFirstname($_POST['firstname']);
+                        $user->setLastname($_POST['lastname']);
+                        $user->setEmail($_POST['email']);
 
-                header("Location: /user/show?id=$userId");
+                        $user->save();
+
+                        header("Location: /user/show?id=$userId");
+                        $mailer->sendMail(($_POST['email']));
```
Comment (danielamoudokpo) [inline www/Controller/User.class.php:88]: ?

File: www/View/verifyaccount.view.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+</head>
+<body>
+
+<?php
+    $msg = '';
+    if(isset($_GET['key']))
+    {
+
+
```
Comment (danielamoudokpo) [inline www/View/verifyaccount.view.php]: to remove

**Review by danielamoudokpo — CHANGES_REQUESTED** [review]

**Review by earvin25 — APPROVED** [review]

**Pull request merged**